compiler/src/iree/compiler/Codegen/Common/ReconcileTranslationInfo.cpp

```diff
@@ -440,10 +635,16 @@ void ReconcileTranslationInfoPass::runOnOperation() {
     // Resolve workgroup distribution related `scf.forall` ops.
     if (failed(resolveWorkgroupForAll(rewriter, rootFuncOp, distributeAlong))) {
       variantOp.emitOpError(
-          "failed in iree-codegen-reconcile-translation-info pass");
+          "failed to resolve workgroup distribution forall ops");
       return signalPassFailure();
     }
 
+    // Resolve split reduction distribution
+    if (failed(
+            resolveSplitReduceForAll(rewriter, rootFuncOp, distributeAlong))) {
+      variantOp.emitOpError("failed to resolve split reduction forall ops");
+    }
```

Comment:
nit: missing a period.

I think you forgot to signal a pass failure here?

```suggestion
    // Resolve split reduction distribution.
    if (failed(
            resolveSplitReduceForAll(rewriter, rootFuncOp, distributeAlong))) {
      variantOp.emitOpError("failed to resolve split reduction forall ops");
    }
```